test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua

```diff
@@ -0,0 +1,108 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- INVALID_DATA
```

Comment:
@OVikhrov, please update Requirement summary and add related to SetMediaClockTimer functional requirement also